tests/runner/testrunner.go

```diff
@@ -61,16 +62,19 @@ func (tr *TestRunner) Start(m runnable) int {
 	err := tr.Platform.setup()
 	defer tr.tearDown()
 	if err != nil {
+		log.Errorf("Failed Platform.setup(), %s", err.Error())
 		return runnerFailExitCode
 	}
 
 	// install components
 	if err := tr.Platform.addComponents(tr.components); err != nil {
+		log.Errorf("Failed Platform.addComponents(), %s", err.Error())
```

Comment:
If just for debugging, revert file changes.

Reply:
I think it's worth keeping this.  It only adds a bit of tracing and helps the user better know how far along the test + this is *fatal* and doesn't log anything to the user today.  When I was first running this it was failing in deployment, and since there were no errors logs around deploy, the output pretty much just had "fail" and nothing else.  What do you think?